src/test/java/com/endava/service/IncomeExpenseServiceImplTest.java

```diff
@@ -234,22 +229,22 @@ public void testViewIncomesMoreExpensiveThanIncorrectAmount() {
 
         Double amount = -50.0;
 
-        incomeExpenseService.viewIncomesMoreExpensiveThan(amount);
+        incomeExpenseService.getMoneyTransferMoreExpensiveThan(amount);
 
-        verify(incomeDao, never()).readAll();
+        verify(moneyTransferDao, never()).readAll();
     }
 
     @Test
     public void testViewIncomesMoreExpensiveThanCorrectAmount() {
-        List<Income> incomes = new ArrayList<>( Arrays.asList(income1, income2, income3, income4) );
+        List<MoneyTransfer> moneyTransfers = new ArrayList<>( Arrays.asList(moneyTransfer, moneyTransfer1, moneyTransfer2, moneyTransfer3) );
         Double amount = 100.0;
 
-        when(incomeDao.readAll()).thenReturn(incomes);
+        when(moneyTransferDao.readAll()).thenReturn(moneyTransfers);
 
-        List<Income> actualIncomes = incomeExpenseService.viewIncomesMoreExpensiveThan(amount);
-        List<Income> expectedIncomes = new ArrayList<>( Arrays.asList(income3, income4) );
+        List<MoneyTransfer> actualIncomes = incomeExpenseService.getMoneyTransferMoreExpensiveThan(amount);
+        List<MoneyTransfer> expectedIncomes = new ArrayList<>( Arrays.asList(moneyTransfer2, moneyTransfer3) );
```

Comment:
You don't need new ArrayList here. Arrays.asList returns unmodifiable List, right what you need :)